mephisto/abstractions/blueprints/static_react_task/static_react_blueprint.py

```diff
@@ -125,3 +130,26 @@ def assert_task_args(
         assert link_task_source == False or (
             link_task_source == True and current_architect in allowed_architects
         ), f"`link_task_source={link_task_source}` is not compatible with architect type: {args.architect._architect_type}. Please check your task configuration."
+
+    def get_frontend_args(self) -> Dict[str, Any]:
+        """
+        Specifies what options within a task_config should be fowarded
+        to the client for use by the task's frontend
+        """
+        # Start with standard task configuration arguments
+        frontend_task_config = super().get_frontend_args()
+        shared_state = self.shared_state
+        assert isinstance(
+            shared_state, SharedTaskState
+        ), "Must use SharedTaskState with StaticReactBlueprint"
+        task_name = self.task_run.to_dict()["task_name"]
+        db = LocalMephistoDB()
+        mephisto_data_browser = MephistoDataBrowser(db)
+        metadata = mephisto_data_browser.get_metadata_from_task_name(task_name)
+        accepted_tips = list(
+            filter(lambda tip: tip["accepted"] == True, metadata["tips"])
+        )
+        frontend_task_config.update({"metadata": {"tips": accepted_tips}})
+        # Use overrides provided downstream
+        frontend_task_config.update(self.frontend_task_config)
+        return frontend_task_config
```

Comment:
Is this the same function as in mephisto/abstractions/blueprints/remote_procedure/remote_procedure_blueprint.py. It's hard tell if they are same or different. If same, does it make sense to refactor into common code.

Reply:
Yeah, I think I thought that this method was implementing an abstract version, but this doesn't actually seem to be the case. I should be able to make it into common code.